main.tf

```diff
@@ -22,41 +22,22 @@ variable "domain" {
   default = "fastfeedback.rocks"
 }
 
-data "aws_route53_zone" "domain" {
-  name = "${var.domain}"
+variable "hackathon_location" {
```

Comment:
This variable name is a bit vague

Reply:
We will change it to inventories_location. We are also going to change around the security groups a bit.